app/site.rb

```diff
@@ -68,10 +68,11 @@ def git?
 			cache_path.join('.git').directory?
 		end
 
-		def status
-			status = @options[:status] || if defined?(DB)
-				job = DB[:jobs].where(site_id: @options[:site_id]).order(:created_at).last
-				job && "build_#{BuildJob.get(job[:job_id]).status}".to_sym
+		def status(db=nil, buildjob=nil)
```

Comment:
set DB and BuildJob contstants in the test, then remove them after the test, instead of this.